sysdeps/linux/generic/sysdeps.cpp

```diff
@@ -747,6 +747,13 @@ int sys_timerfd_create(int clockid, int flags, int *fd) {
 	return 0;
 }
 
+int sys_timerfd_settime(int fd, int flags, const struct itimerspec *value, const struct itimerspec *oldvalue) {
+	auto ret = do_syscall(SYS_timerfd_settime, fd, flags, value, oldvalue);
```

Comment:
Do we want to use the 32 bit variant here?

Reply:
What exactly do you mean by '32 bit variant'?

Reply:
Ah, wait, never mind. There is a `SYS_timerfd_settime64` syscall but it only exists on 32-bit architectures.